tests/test_zuora_discovery.py

```diff
@@ -1,36 +1,38 @@
-from tap_tester import menagerie, connections
+from base import ZuoraBaseTest
+from tap_tester import connections, menagerie
 from tap_tester.logger import LOGGER
 
-from base import ZuoraBaseTest
 
 class DiscoveryTest(ZuoraBaseTest):
-    """Test tap discovery mode and metadata/annotated-schema conforms to standards."""
+    """Test tap discovery mode and metadata/annotated-schema conforms to
+    standards."""
 
     @staticmethod
     def name():
         return "tap_tester_zuora_discovery_rest"
 
     def test_run(self):
-        """ Executing tap-tester scenarios for both types of zuora APIs AQUA and REST"""
+        """Executing tap-tester scenarios for both types of zuora APIs AQUA and
+        REST."""
         self.run_test("AQUA")
         self.run_test("REST")
 
     def run_test(self, api_type):
-        """
-        Verify that discover creates the appropriate catalog, schema, metadata, etc.
-
-        • Verify number of actual streams discovered match expected
-        • Verify the stream names discovered were what we expect
-        • Verify stream names follow naming convention
-          streams should only have lowercase alphas and underscores
-        • verify there is only 1 top level breadcrumb
-        • Verify primary key(s) match expectations.
-        • Verify replication key(s) match expectations.
-        • verify that if there is a replication key we are doing INCREMENTAL otherwise FULL
-        • verify the actual replication matches our expected replication method
-        • verify that primary, replication and foreign keys
-          are given the inclusion of automatic (metadata and annotated schema).
-        • verify that all other fields have inclusion of available (metadata and schema)
+        """Verify that discover creates the appropriate catalog, schema,
+        metadata, etc.
+
+        • Verify number of actual streams discovered match expected •
+        Verify the stream names discovered were what we expect • Verify
+        stream names follow naming convention   streams should only have
+        lowercase alphas and underscores • verify there is only 1 top
+        level breadcrumb • Verify primary key(s) match expectations. •
+        Verify replication key(s) match expectations. • verify that if
+        there is a replication key we are doing INCREMENTAL otherwise
+        FULL • verify the actual replication matches our expected
+        replication method • verify that primary, replication and
+        foreign keys   are given the inclusion of automatic (metadata
+        and annotated schema). • verify that all other fields have
+        inclusion of available (metadata and schema)
```

Comment:
Mis-alignment here as well. Looks like `•` char is causing issue. Use dash instead.